Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -1445,10 +1445,10 @@ def _addAxis(self, axisObject):
             for label in axisObject.axisLabels:
                 self._addAxisLabel(labelsElement, label)
             axisElement.append(labelsElement)
-        if isinstance(axisObject, AxisDescriptor):
+        if hasattr(axisObject, "minimum"):
             axisElement.attrib['minimum'] = self.intOrFloat(axisObject.minimum)
             axisElement.attrib['maximum'] = self.intOrFloat(axisObject.maximum)
-        elif isinstance(axisObject, DiscreteAxisDescriptor):
+        elif hasattr(axisObject, "values"):
             axisElement.attrib['values'] = " ".join(self.intOrFloat(v) for v in axisObject.values)
         axisElement.attrib['default'] = self.intOrFloat(axisObject.default)
```

Comment:
This fails for `DiscreteAxisDescriptor` which does not have a default value. (Also not sure why discrete axes would not want a default value, but I did not follow any discussions about that)

Reply:
`DiscreteAxisDescriptor` has a default value: https://fonttools.readthedocs.io/en/latest/designspaceLib/python.html#fontTools.designspaceLib.DiscreteAxisDescriptor.default

Reply:
(traceback posted)